### PR TITLE
Updated README for issue#294

### LINKED ISSUE #294: Mention critique in README
Łukasz Langa (@ambv), the original author of [Black](https://github.com/psf/black) has [written in his blog](https://lukasz.langa.pl/36380f86-6d28-4a55-962e-91c2c959db7a/) in July 2021 about why Black insists on formatting an entire code base instead of just parts of it.

The article has very good points which developers should probably consider before opting to use Darker (or a similar partial formatting method) for their projects.

Maybe we should mention this in Darker's README?

### PR DESCRIPTION
**Pull request by dhrvjha**
Added link to original article and quoted the original text from the article
Fixes #294

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -98,6 +98,10 @@ __ https://github.com/psf/black/issues/1352
 But for the time being, this is where ``darker`` enters the stage.
 This tool is for those who want to do partial formatting right now.
 
+> "When you make this single reformatting commit, everything that comes after is semantic changes so your commit history is clean ...
+However, by doing that you forever expose yourself to commits that are a mix of semantic changes with stylistic changes."
+>  <cite>[Łukasz Langa](https://lukasz.langa.pl/)</cite> - creator of Black, [article](https://lukasz.langa.pl/36380f86-6d28-4a55-962e-91c2c959db7a/)
+
```
Comment (akaihola) [inline README.rst:104]: This doesn't seem to render well on GitHub:
image

**Comment by akaihola** [comment]
Hi @dhrvjha,
I'd still like to return to this pull request a bit later, complete it and get it merged. Do you mind if I reopen it and return to it after I've dealt with a couple of other PRs first?

**Comment by akaihola** [comment]
Reopening, I have some changes to push.

**Comment by akaihola** [comment]
@dhrvjha, could you review my changes to the README additions?

**Comment by akaihola** [comment]
@dhrvjha, @ambv, or @jack1142, could you review my changes to the README additions?

**Comment by jack1142** [comment]
Hmm, it might get a bit long but personally I feel like it's important to mention that you can avoid reformat commit showing in git blame by using ignore revs file (which GitHub now supports btw) which is explained more comprehensively here:
https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html

**Comment by akaihola** [comment]
@jack1142, thanks for the idea, I added a mention of `ignoreRevsFile` and linked to Git documentation.

**Pull request merged**